modules/container-service/managed-clusters/main.bicep

```diff
@@ -416,6 +425,12 @@ resource managedCluster 'Microsoft.ContainerService/managedClusters@2022-11-01'
     agentPoolProfiles: primaryAgentPoolProfile
     linuxProfile: (empty(aksClusterSshPublicKey) ? null : aksClusterLinuxProfile)
     servicePrincipalProfile: (empty(aksServicePrincipalProfile) ? null : aksServicePrincipalProfile)
+    ingressProfile: {
+      webAppRouting: {
+        enabled: webApplicationRoutingEnabled
+        dnsZoneResourceId: !empty(dnsZoneResourceId) ? any(dnsZoneResourceId) : null
+      }
+    }
```

Comment:
@samuelya can you please provide some test parameters for this change so we can validate the workflow in case someone would like to use these parameters. We have a couple of AKS test deployments [here](https://github.com/Azure/ResourceModules/tree/main/modules/container-service/managed-clusters/.test). You can select either the Azure or the kubenet version. This requires you to create a private DNS zone in the dependencies which then can be referenced in the module deployment. Happy to support you in this if you need more guidance.